Tests for the classes Bun, Burger, Ingredients

gestia535 opened the pull request:
Added tests for the following classes: Bun, Burger, Ingredients, Database.

Added allure report, coverage report.
Added requirements.txt

gestia535 committed:
Added tests for the following classes: Bun, Burger, Ingredients, Database.
Added allure report, coverage report.
Added requirements.txt

AlexKitIt commented on conftest.py:
```diff
@@ -0,0 +1,54 @@
+from praktikum.ingredient import Ingredient
+
+
+@pytest.fixture
+def burger():
+    return Burger()
```
Нужно исправить: слишком примитивный объект чтобы быть предусловием. Если он не наполняется, то создаваться должен где используется

AlexKitIt commented on conftest.py:
```diff
@@ -0,0 +1,54 @@
+    return bun
+
+
+@pytest.fixture
+def database():
+    return Database()
```
up

gestia535 committed:
Исправления после ревью:
Фикстуры burger и database были удалены.
В файлы test_burger.py и test_database.py были внесены строчки инициализирующие объекты burger и database.
